drivers/source/PwmOut.cpp

```diff
@@ -98,6 +98,19 @@ void PwmOut::pulsewidth_us(int us)
     core_util_critical_section_exit();
 }
 
+void PwmOut::enable_pwm(bool enabled)
```

Comment:
Do we actually need to `pwmout_free` to free the HW (and then to restore the state)? I'm not sure what will happen with HW state if the high frequency clock would go down and also HW may limit the low power state based on active HW.

Reply:
https://github.com/ARMmbed/mbed-os/pull/11641#issuecomment-539477808